compiler/luci/import/include/luci/Import/GraphBuilderRegistry.h

```diff
@@ -32,6 +33,11 @@ struct GraphBuilderSource
    * @brief Returns registered GraphBuilder pointer for operator (nullptr if not present)
    */
   virtual const GraphBuilderBase *lookup(const circle::BuiltinOperator &op) const = 0;
+
+  /**
+   * @brief Returns registered NodeBuilderBase pointer for type (nullptr if not present)
+   */
+  virtual const NodeBuilderBase *lookup(NodeBuilderType type) const = 0;
```

Comment:
Can we use `const NodeBuilderType type` ?

Reply:
Yes, we can!

Reply:
Done!